init-tools.sh

```diff
@@ -42,7 +43,7 @@ esac
 
 if [ ! -e $__PROJECT_JSON_FILE ]; then
     if [ -e $__TOOLRUNTIME_DIR ]; then rm -rf -- $__TOOLRUNTIME_DIR; fi
-
+    echo "Running: $__scriptpath/init-tools.sh" > $__init_tools_log
```

Comment:
Consider adding $* so that it echoes arguments, if there are any:
`echo "Running: $__scriptpath/init-tools.sh $*" > $__init_tools_log`